nautobot/docs/configuration/index.md

```diff
@@ -6,11 +6,11 @@ This section describes how to get started with configuring Nautobot.
 
 An initial configuration can be created by executing `nautobot-server init`. This will generate a new configuration with all of the default settings provided for you, and will also generate a unique [`SECRET_KEY`](required-settings.md#secret_key).
 
-By default the file will be created at `$HOME/.nautobot/nautobot_config.py`:
+By default (if you haven't set [`NAUTOBOT_ROOT`](#nautobot-root-directory) to some other value), the file will be created at `$HOME/.nautobot/nautobot_config.py`:
 
 ```no-highlight
 $ nautobot-server init
-Configuration file created at '/home/example/.nautobot/nautobot_config.py'
+Configuration file created at '/opt/nautobot/nautobot_config.py'
```

Comment:
Revert this?

Reply:
I reverted it and then added a tip, explaining that the installation docs set `NAUTOBOT_ROOT` to `/opt/nautobot`.  Otherwise, it gets very confusing because installation docs use `/opt/nautobot` but then in some areas like this, we seemingly randomly revert back to the default location.